moon/docker/rover/Dockerfile

```diff
@@ -5,13 +5,23 @@ SHELL ["/bin/bash", "-c"]
 RUN apt-get update && apt-get install -y python-pip python3-pip openssh-client vim less
 RUN python -m pip install pyzmq
 
-RUN /usr/bin/python3 -m pip install --user --no-cache-dir msgpack "opencv-python>3.4.2,<4" pyzmq shapely scipy
+RUN /usr/bin/python3 -m pip install scikit-build
```

Comment:
the motivation for a single line with one `RUN` is that it creates only one layer in docker (and the number of layers is limited) ... I suppose that your motivation was readability or something else?

Reply:
I see number of processors for opencv ...

Reply:
No, this was required to build at all. --user (ie install into user space instead of system) was not adequate for NASA submission build. Removing it and building into system started requiring scikit-build. That had to be installed before the other package could be built, hence two separate lines.